packages/feedback/components/feedback-container/src/feedback-container.template.ts

```diff
@@ -0,0 +1,255 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2023
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { LitElement, html, css } from 'lit';
+import { property, state } from 'lit/decorators.js';
+
+import '@carbon/web-components/es/components/textarea/index.js';
+import '@carbon/web-components/es/components/radio-button/index.js';
+import '@carbon/web-components/es/components/button/index.js';
+
+export class FeedbackContainer extends LitElement {
+  static styles = css`
```

Comment:
can these styles be separated out into its own file? (see extended-button component for example)